__tests__/app.test.js

```diff
@@ -273,3 +273,47 @@ describe("9. GET /api/users", () => {
       });
   });
 });
+
+describe("10. GET /api/articles (queries)", () => {
+  test("Returns An article response object should also now include", () => {
```

Comment:
this test description isn't really saying what we're testing for here - in this case you're testing for an existing topic sorted in descending order